pheweb/serve/server_jeeves.py

```diff
@@ -430,11 +447,18 @@ def chip(self):
     @functools.lru_cache(None)
     def get_gene_region_mapping(self):
         return {genename: (chrom, pos1, pos2) for chrom, pos1, pos2, genename in get_gene_tuples()}
-
+    
     @functools.lru_cache(None)
-    def get_best_phenos_by_gene(self):
-        with open(common_filepaths['best-phenos-by-gene']) as f:
-            return json.load(f)
+    def get_best_phenos_by_gene(self, gene):
+        # get top variants for the gene
+        if 'best-phenos-by-gene' in common_filepaths:
```

Comment:
This if can now be removed? Looks like the function is returning two different kinds of things depending on whether best-phenos-by-gene is there or not

Reply:
You are right. I removed now the part with previous implementation, how does it look now?